### PR TITLE
refactor: refactor TestStoreAndInstantiateContract

### PR DESCRIPTION
**Pull request by kokeshiM0chi**
## Description

Refactor TestStoreAndInstantiateContract

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/Finschia/wasmd/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/Finschia/wasmd/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [ ] I have added tests to cover my changes. (not needed)
- [ ] I have updated the documentation accordingly. (not needed)
- [ ] I have updated API documentation `client/docs/swagger-ui/swagger.yaml` (not needed)

### CODE REVIEW

File: x/wasmplus/keeper/msg_server_integration_test.go [hunk 1]
```diff
@@ -69,23 +111,7 @@ func TestStoreAndInstantiateContract(t *testing.T) {
...
-			assert.Equal(t, 2, len(events[1].Attributes))
-			assert.Equal(t, "module", string(events[1].Attributes[0].Key))
-			assert.Equal(t, "wasm", string(events[1].Attributes[0].Value))
-			assert.Equal(t, "sender", string(events[1].Attributes[1].Key))
-			assert.Equal(t, "instantiate", events[2].Type)
-			assert.Equal(t, "_contract_address", string(events[2].Attributes[0].Key))
-			assert.Equal(t, storeAndInstantiateResponse.Address, string(events[2].Attributes[0].Value))
-			assert.Equal(t, "code_id", string(events[2].Attributes[1].Key))
-			assert.Equal(t, "1", string(events[2].Attributes[1].Value))
+			assert.True(t, reflect.DeepEqual(spec.events, rsp.Events))
```
Comment (170210) [inline x/wasmplus/keeper/msg_server_integration_test.go:114]: Please check https://github.com/Finschia/wasmd/pull/76#discussion_r1298307663

File: x/wasmplus/keeper/msg_server_integration_test.go [hunk 2]
```diff
@@ -2,14 +2,16 @@ package keeper_test
...
 	"testing"
 	"time"
 
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
+	abcitypes "github.com/tendermint/tendermint/abci/types"
 	tmproto "github.com/tendermint/tendermint/proto/tendermint/types"
 
-	sdk "github.com/Finschia/finschia-sdk/types"
+	sdktypes "github.com/Finschia/finschia-sdk/types"
```
Comment (170210) [inline x/wasmplus/keeper/msg_server_integration_test.go:14]:
```suggestion
	sdk "github.com/Finschia/finschia-sdk/types"
```

File: x/wasmplus/keeper/msg_server_integration_test.go [hunk 3]
```diff
@@ -25,18 +27,58 @@ func TestStoreAndInstantiateContract(t *testing.T) {
...
+				},
+			}, {
+				Type: "message",
+				Attributes: []abcitypes.EventAttribute{abcitypes.EventAttribute{
+					Key:   []byte("module"),
+					Value: []byte("wasm"),
+					Index: false,
+				}, {
+					Key:   []byte("sender"),
+					Value: []byte("link1qqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqqjhjmun"),
```
Comment (170210) [inline x/wasmplus/keeper/msg_server_integration_test.go:62]: Please use `myAddress.String()`

**Review by 170210 — COMMENTED** [review]
I think there's no need to test the `Index` field

File: x/wasmplus/keeper/msg_server_integration_test.go (hunk 3 above)
Comment (kokeshiM0chi) [reply x/wasmplus/keeper/msg_server_integration_test.go]: Thanks. I fixed it as pointed out.

File: x/wasmplus/keeper/msg_server_integration_test.go (hunk 1 above)
Comment (kokeshiM0chi) [reply x/wasmplus/keeper/msg_server_integration_test.go]: Thanks. I fixed it as pointed out.

**Comment by da1suk8** [comment]
LGTM, I will approve when the conflict has been corrected.

**Comment by kokeshiM0chi** [comment]
@170210 Since there is no particular reason to delete it, it is better to test all fields.

**Review by dudong2 — DISMISSED** [review]

**Review by da1suk8 — APPROVED** [review]

File: x/wasmplus/keeper/msg_server_integration_test.go [hunk 4]
```diff
@@ -67,25 +96,27 @@ func TestStoreAndInstantiateContract(t *testing.T) {
...
+// This function is utilized to generate the msg event for event checking in integration tests
+// It will be deleted in release/v0.1.x
+func createMsgEvent(sender sdk.AccAddress) abci.Event {
+	return abci.Event{
+		Type: "message",
+		Attributes: []abci.EventAttribute{
+			{
+				Key:   []byte("module"),
+				Value: []byte("wasm"),
+			},
```
Comment (170210) [inline x/wasmplus/keeper/msg_server_integration_test.go:115]: If you want to test all fields, please add the Index field

File: x/wasmplus/keeper/msg_server_integration_test.go [hunk 5]
```diff
@@ -67,25 +96,27 @@ func TestStoreAndInstantiateContract(t *testing.T) {
...
+			{
+				Key:   []byte("sender"),
+				Value: []byte(sender.String()),
+			},
```
Comment (170210) [inline x/wasmplus/keeper/msg_server_integration_test.go:119]: The same as above comment

File: x/wasmplus/keeper/msg_server_integration_test.go (hunk 4 above)
Comment (kokeshiM0chi) [reply x/wasmplus/keeper/msg_server_integration_test.go:115]: Thanks. That's right.

File: x/wasmplus/keeper/msg_server_integration_test.go (hunk 5 above)
Comment (kokeshiM0chi) [reply x/wasmplus/keeper/msg_server_integration_test.go:119]: I fixed both. Thanks

**Comment by kokeshiM0chi** [comment]
Sorry, the last two commits are wrong, so I will force push.

**Review by 170210 — APPROVED** [review]

**Review by da1suk8 — APPROVED** [review]

**Pull request merged**